client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { trim, isEmpty } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Gravatar from 'components/gravatar';
+import FollowButton from 'reader/follow-button';
+import Gridicon from 'gridicons';
+import { getStreamUrl } from 'reader/route';
+
+const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( '/', '' );
+
+function SettingsMenu() {
+	return (
+		<span className="reader-subscription-list-item__settings-menu">
+			<Gridicon
+				icon="cog"
+				size={ 20 }
+			/>
+			Settings
+		</span>
+	);
+}
+
+function ReaderSubscriptionListItem( {
+	isFollowing,
+	siteUrl,
+	siteTitle,
+	siteAuthor,
+	siteExcerpt,
+	feedId,
+	siteId,
+	className = '',
+	onSiteClick = () => {},
+	followSource,
+	lastUpdated,
+	translate,
+} ) {
+	// prefer a users name property
+	// if that doesn't exist settle for combining first and last name
+	const authorName = siteAuthor.name ||
+		trim( `${ siteAuthor.first_name || '' } ${ siteAuthor.last_name || '' }` );
+	const readerStreamUrl = getStreamUrl( feedId, siteId );
+
+	return (
+		<div className={ classnames( 'reader-subscription-list-item', className ) }>
+			<div>
+				<a href={ siteUrl } onClick={ onSiteClick }>
+					<Gravatar user={ siteAuthor } />
+				</a>
+			</div>
+			<div className="reader-subscription-list-item__byline">
```

Comment:
@samouri a context is used to differentiate two translatable strings with a different meaning. For example "Post" (verb) and "Post" (noun).
It creates a separate instance of the string for our translators to translate.

If you just want to clarify the meaning of the string, you can use a translator comment instead (`comment` instead of `context` in the `translate` args)